scoring/templates/scoring/section.html

```diff
@@ -414,33 +390,28 @@ <h5 class="modal-title" id="section-{{ forloop.parentloop.counter }}-question-{{
                           </div>
                       </td>
 
+                      {% for comparison in question.comparisons %}
                       {% comment %} On mobile we will display only the average for the type of council. {% endcomment %}
                       <td data-column="score" class="text-start text-md-end d-flex flex-row d-md-table-cell score">
-                        {% comment %} TODO add score {% endcomment %}
-                          <span class="me-2">1/3</span>
-                          <span class="badge bg-secondary d-md-none">Boerset Council</span>
+                        <span class="me-2">{{ comparison.score }}/{{ question.details.max_score }}</span>
+                        <span class="badge bg-secondary d-md-none">{{ comparison.council }}</span>
```

Comment:
In here is not pulling the name of the council. This is the mobile view just in case.

Reply:
ooops, fixed.